docs/content/testing/writing-a-new-test.md

```diff
@@ -0,0 +1,314 @@
+{
+    [TestFixture]
+    public class MyClassStaticTests
+    {
+    }
+
+    [TestFixture]
+    public class MyClassInstanceTests
+    {
+        private MyClass myClassInstance;
+
+        [SetUp]
+        public void SetUp()
+        {
+            myClassInstance = new MyClass();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            myClassInstance.Dispose();
+        }
+    }
+}
+```
+
+### Test methods
+
+The following are some example test methods. Note that the test should ensure that the `AddTwoNumbers` static method returns the sum of the
+ input numbers.
+
+The test name should match this expectation and contain relevant assertions.
+
+```cs
+[TestFixture]
+public class MyClassStaticTests
+{
+    [Test]
+    public void AddTwoNumbers_should_return_the_sum_of_the_input_numbers()
+    {
+        Assert.AreEqual(4, MyClass.AddTwoNumbers(1, 3));
+        Assert.AreEqual(4, MyClass.AddTwoNumbers(3, 1));
+        Assert.AreEqual(7, MyClass.AddTwoNumbers(2, 5));
+    }
+}
+```
+
+For the instance functionality, you can add tests similarly:
+
+```cs
+[TestFixture]
+public class MyClassInstanceTests
+{
+    private MyClass myClassInstance;
+
+    [SetUp]
+    public void SetUp()
+    {
+        myClassInstance = new MyClass();
+    }
+
+    [TearDown]
+    public void TearDown()
+    {
+        myClassInstance.Dispose();
+    }
+
+    [Test]
+    public void InstanceValue_is_0_by_default()
+    {
+        Assert.AreEqual(0, myClassInstance.InstanceValue);
+    }
+
+    [Test]
+    public void DoInstanceOperation_increases_InstanceValue_by_1()
+    {
+        myClassInstance.DoInstanceOperation();
+        Assert.AreEqual(1, myClassInstance.InstanceValue);
+
+        myClassInstance.DoInstanceOperation();
+        Assert.AreEqual(2, myClassInstance.InstanceValue);
+    }
+
+    [Test]
+    public void DoSomethingElse_increses_InstanceValue_by_5()
+    {
+        myClassInstance.DoSomethingElse();
+        Assert.AreEqual(5, myClassInstance.InstanceValue);
+
+        myClassInstance.DoSomethingElse();
+        Assert.AreEqual(10, myClassInstance.InstanceValue);
+    }
+}
+```
+
+### Internal variables and methods
+
+The tests above will not compile. This is because the test class is in a different assembly to the implementation class, and the tests are looking at internal fields.
+
+To solve this; make the internals of the the `MyClass.cs` file visible to the test assembly in the way shown below.
+
+```
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Improbable.Gdk.Core.EditModeTests")]
+```
+
+### Running the new tests
+
+In the Test Runner window of Unity, find your test under: **Improbable.Gdk.Core.EditModeTests.dll**. (This matches the folder name
+ `Gdk/Core/Tests/EditMode`.)
+
+You can follow the namespace and fixture name in the hierarchy:
+
+- **Improbable** > ** Gdk**  > ** Core**  > ** EditModeTests** > ** Utility**  > ** MyClassStaticTests** 
```

Comment:
the spaces between ** and the words are breaking your bolding here

Reply:
Nice catch! Fixed